### ISSUE TITLE
Ajouter les équipements "stables" & les afficher sur le graphique

### ISSUE THREAD

**Issue by Clem-mrtn** [datagir/wattris#32]
> En tant qu'utilisateur je veux pouvoir comprendre quels équipements consomment et quels équipements sont plus anecdotiques

Pour les équipements dont la consommation est stable : 
- frigo 
- box 
- congélateur 
=> trouver une façon de les afficher pour faire passer le message qu'il n'y pas de pic de puissance / consommation, sans que l'utilisateur ne comprennent que ces appareils consomment peu

**Comment by Clem-mrtn** [comment]
- Pour la v1 ajouter les petits appareils dans le .json

**Comment by Clemog** [comment]
Suite appel Florian:

- Traiter de la même laa même manière tous les appareils dans une V1
- Travailler sur un bouton "toute la journée"
- Gérer la couleur des appareils 24h
- Trouver un ordre d'affichage convenable (pas la box TV en 2 sur la liste)

**Comment by Clemog** [comment]
> * revoir la liste : par ordre alphabétique si on ne fait pas de distinction ou alors réfléchir à une nouvelle UI pour les séparer

Pourquoi ne pas proposer un ordre que l'on définit ?



> * Quid du chargeur de portable ?

C'est à dire ? 



> * Attention, quand on ajoute un appareil **toute la journée** cela prend en compte l'heure affichée par défaut dans l'onglet "Je le lance à" alors qu'on est pas forcément passé par l'onglet. On se retrouve donc avec un pic d'allumage qu'on a pas choisi sur le graphique (ex les radiateurs lancés à 18h00)

Oui, il y a un travail de design sur cet "affichage 24h" qui est bien un mode d'ajout spécifique. Et c'est également noté qu'il faut travailler, à priori, spécifiquement sur le parcours d'ajout du chauffage. Tu vois un autre exemple d'appareil qui pourrait poser problème ? (Utilisatation 24h avec pic d'allumage)

> * Est-ce qu'on peut sélectionner **toute la journée** pour tous les appareils ?

Pas sur (revient à la réflexion du point précéden :) )

**Comment by Clemog** [comment]
> Je me pose aussi une question sur la compréhension : dans le cas présent ci-dessous quand j'ajoute par exemple ces 3 appareils : lave-linge, plaques & radiateur électrique mon score consommation est rouge en mode saturation du réseau mais toutes mes cartes en dessous sont **bleues** ?
> Hors d'ordinaire quand je suis en tension sur le réseau ma carte devient rouge pour distinguer quel appareil pose problème

Yes ça regroupe les points précédent sur le parcours chauffage et la gestion du "24h"

**Comment by florianpanchout** [comment]
On pourrait simplifier en utilisation toute la journée => pas de pic d'allumage non ?

(le chauffe-eau étant un cas particulier que je propose de ne pas traiter dans cette v1)

**Comment by Clemog** [comment]
Le chauffe eau c'est finalement 8h (je me suis planté, il chauffe (normalement) que pendant les heures creuses ou en tout cas pdt 8h 
Et je pense que le chauffage est le seul appareil qui pose problème

**Comment by Clem-mrtn** [comment]
Quelques nouveaux retours sur l'ajouts de petits appareils : 
- Frigo combiné je ne suis pas sûre que ce soit très bien compris ?
- Traduire les anglicismes: Charge Laptop => **charge de l'ordinateur portable**

**Comment by Clemog** [comment]
C'est bon :)
